tenants-certified-attributes-updater/src/main/scala/it/pagopa/interop/tenantscertifiedattributesupdater/util/Utils.scala

```diff
@@ -96,7 +96,11 @@ object Utils {
         }
 
     val revocations: Map[PersistentExternalId, List[AttributeInfo]] =
-      fromTenant.toList
+      fromTenant
+        .filter { case (_, attrs) =>
+          attrs.exists(_.revocationTimestamp.isEmpty)
```

Comment:
[this point](https://github.com/pagopa/interop-be-jobs/pull/146#discussion_r1277311182) is still true.
If you prefer, attributes are unpacked at line 104, you can filter by revocationTimestamp after that

Reply:
I cannot understand why this filter at line 104 is more efficient than at line 99.